submissions/opeyemiPeter/challenge1-arrays/question1.js

```diff
@@ -0,0 +1,24 @@
+//  Question one
+
+function solution1(strs) {
+  let toBeReturned = "";
+  for (let i = 1; i < strs[0].length; i++) {
```

Comment:
Hello @opeyemipeter1759. Nice Idea. 
I think it works generally. 
However, because your loop starts at index 1, it wouldn't work for a case where the array contains a single element.
it would return `toBeReturned` which is instantiated to an empty string.

Reply:
setting the loop to 1 was an error I just noticed, I was to set it to 0.